### ISSUE TITLE
Why is not published this app in Google Play?

### ISSUE THREAD

**Issue by billyjoker** [vidit135g/Sticker-Central#2]
Hi, i'd like to know if you tried to publish your app in google play and if you had any problem with their policies, because you have some categories laike Homer Simpson and Game of Thrones. You have a great stuff!!

**Comment by vidit135g** [comment]
You are right, btw sorry for the late reply. There was an issue in some of the stickers which there was a violation of one of the policies for

**Comment by vidit135g** [comment]
There was a violation due to one of the stickers of Harley Quinn and what they did, they straight away suspended my developer account without any further notice aur explanation, I tried to contact them so many times but they didn't respond. Because of this, my other apps were also gone.

**Comment by billyjoker** [comment]
Oh that's terrible, from the last years they are very straight with their policies. Did you try to contact them with their live chat? Sometimes you get a second revision of your case chatting with a person from Google:

https://support.google.com/googleplay/android-developer/contact/pre_chat_survey?hl=en&cfnti=escalationflow.chat&cft=1

I suppose you also tried with the appeal form:

https://support.google.com/googleplay/android-developer/contact/appappeals

**Comment by vidit135g** [comment]
It's completely apt for being strict with the policies but according to them some of my stickers were "sexually" explicit 😅and despite changing the rating of my app amd even deleting all those stickers from my app and introducing new updated app, I still got my account deleted 😂. The main problem is there are n number of apps which have far more sexually explicit content and they break all the Google policies but are easily running from past so many years and are having large number of downloads. Unfortunately, they only saw my account to deal with 😅😂.

**Comment by billyjoker** [comment]
Yes i know that behaviour from Google. When you say that there are n number of apps which have sexually explicit content, very probably that apps have not been updated for years, when the policies were not so tough. If one of them is currently updated, very probably get a suspension as well.

**Comment by vidit135g** [comment]
Yeah, that's true. Btw, I'm planning to buy a new one very soon.
